test/operator/test_operator.py

```diff
@@ -186,13 +190,22 @@ def test_is_hermitean(op):
     hi = op.hilbert
     assert len(hi.local_states) == hi.local_size
 
+    def _sort_get_conn(op, s):
+        sp, mels = op.get_conn(s)
+        nbp = op.hilbert.states_to_numbers(sp)
+        _nbp = np.where(mels != 0, nbp, np.max(nbp) + 1)
+        p = np.argsort(_nbp)
+        sp = op.hilbert.numbers_to_states(np.take_along_axis(nbp, p, axis=-1))
+        mels = np.take_along_axis(mels, p, axis=-1)
+        return sp, mels
+
     rstates = hi.random_state(rng.next(), 100)
     for i in range(len(rstates)):
         rstate = rstates[i]
-        rstatet, mels = op.get_conn(rstate)
+        rstatet, mels = _sort_get_conn(op, rstate)
 
```

Comment:
The test failure is because he is testing the connected elements of the elements with matrix element 0, which breaks some assumptions...

I would change this to that after limited testing seems to work. This simply avoids checking zeros, assuming that the converse is also zero.

The 1e-8 is a bit arbitrary but we don't have a common interface for the tolerance of the operators so I think this will do for now.

```python
    rstates = hi.random_state(rng.next(), 100)
    for i in range(len(rstates)):
        rstate = rstates[i]
        rstatet, mels = _sort_get_conn(op, rstate)

        for k, state in enumerate(rstatet):
            invstates, mels1 = _sort_get_conn(op, state)
            # Ignore matrix elements which are below threshold, as they are
            # hard zeros
            # the constant could be tuned
            if np.abs(mels[k]) < 1e-8:
                continue 

            found = False
            for kp, invstate in enumerate(invstates):
                if np.abs(mels1[kp]) < 1e-8:
                    continue 

                if np.array_equal(rstate, invstate.flatten()):
                    found = True
                    np.testing.assert_allclose(mels1[kp], np.conj(mels[k]))
                    break

            assert found
```

Reply:
yeah, that's even better I think. Thank you!